generator/src/test/java/com/scottlogic/deg/generator/reducer/ConstraintReducerTest.java

```diff
@@ -892,6 +893,7 @@ void whenHasStringRestrictions_shouldOnlyFilterStringsInSet() {
         Assert.assertThat(spec.getWhitelist().set(), containsInAnyOrder("lorem", "ipsum"));
     }
 
+    @Disabled("should we have a way of enforcing types on set constraints outside the profile reader")
```

Comment:
When the InSetConstraint is created it should be validated to make sure that all the values in the set are valid for the field type. Once that check is added this test can be removed.

Reply:
this fails hundreds of tests, i'd say it's too big a scope for this ticket